sonar-plugin-api/src/main/java/org/sonar/api/batch/CheckProject.java

```diff
@@ -23,7 +23,9 @@
 
 /**
  * @since 1.10
+ * @deprecated since 5.6
```

Comment:
Nothing. It was a technical interface anyway, shared by Decoratos, old Sensor, Initializer and old PostJob